website/docs/language/settings/backends/s3.mdx

```diff
@@ -352,6 +358,32 @@ The following configuration is optional:
 * `dynamodb_endpoint` - (Optional) **Deprecated** Custom endpoint for the AWS DynamoDB API. This can also be sourced from the `AWS_DYNAMODB_ENDPOINT` environment variable.
 * `dynamodb_table` - (Optional) Name of DynamoDB Table to use for state locking and consistency. The table must have a partition key named `LockID` with type of `String`. If not configured, state locking will be disabled.
 
+### S3 State Locking
+
+* `use_lockfile` - (Optional) Enable locking directly into the configured bucket for the state.
+
+To migrate from DynamoDB to S3 locking, the following steps can be followed:
+1. The new attribute `use_lockfile=true` can be added alongside `dynamodb_table`:
+   * With both attributes specified, OpenTofu will try to acquire the lock first in S3 and if successful, will try to acquire the lock in DynamoDB. In this case, the lock will be considered acquired only when both (S3 and DynamoDB) locks were acquired successfully.
+   * Later, after a baking period with both locking mechanisms enabled, if no issues encountered, remove the `dynamodb_table` attribute. Now, you are solely on the S3 locking.
+   * **Info:** Letting both locking mechanisms enabled, ensures that nobody will acquire the lock regardless of having or not the latest configuration.
```

Comment:
Letting -> Keeping

Reply:
Fixed in 8c9faf7fca2641b352f9eae9127feee2b5901a20